README.md

```diff
@@ -122,32 +119,24 @@ func main() {
 	//	"foo":"bar",
 	//	"count":17
 	//}
+```
 
-	// Advanced use case: wrap a custom core with ecszap core
-	// create your own non-ECS core using a ecszap JSONEncoder
-	encoder := ecszap.NewJSONEncoder(ecszap.NewDefaultEncoderConfig())
-	core = zapcore.NewCore(encoder, os.Stdout, zap.DebugLevel)
-	// wrap your own core with the ecszap core
-	logger = zap.New(ecszap.WrapCore(core), zap.AddCaller())
-	defer logger.Sync()
-	logger.With(zap.Error(errors.New("wrapCore"))).Error("boom")
-	// Log Output:
-	//{
-	//	"log.level":"error",
-	//	"@timestamp":1584716847524082,
-	//	"log.origin":{
-	//		"file.name":"main/main.go",
-	//		"file.line":338
-	//	},
-	//	"message":"boom",
-	//	"ecs.version":"1.5.0",
-	//	"error":{
-	//		"message":"wrapCore"
-	//	}
-	//}
-}
+### Advanced Use Case
```

Comment:
```suggestion
### Wrap a custom underlying zapcore.Core
```

Just an idea. "Advanced User Case" by itself require further reading to understand what exactly it is.